src/backend/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# pull official base image
+FROM python:3.11-slim-buster
```

Comment:
I wonder if there's a way to keep this in sync with the one in pyproject.toml...

Reply:
I mean they're both Python 11, what else can I do?